code/datums/emergency_calls/sectoid.dm

```diff
@@ -0,0 +1,45 @@
+//Sectoids
+/datum/emergency_call/sectoid
+	name = "Sectoids"
+	probability = 5
+	spawn_type = /mob/living/carbon/human/species/sectoid
+	shuttle_id = "distress_ufo"
+
+/datum/emergency_call/sectoid/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<B>You are a sectoid, a mass-cloned alien soldier and psionics expert.")
+	to_chat(H, "<B>You were sent to eliminate the local humans and to scout this sector of space for an abundance of resources. A distress signal from a primitive spacecraft has been picked up by our scanners.</B>")
+	to_chat(H, "<B>Your mission is simple: Destroy all humans, and any other race that poses a threat.</b>")
+
+
+/datum/emergency_call/sectoid/create_member(datum/mind/M)
+	. = ..()
+	if(!.)
+		return
+
+	var/mob/original = M.current
+	var/mob/living/carbon/human/H = .
+	H.name = GLOB.namepool[/datum/namepool/sectoid].random_name(H)
+	H.real_name = H.name
+
+	M.transfer_to(H, TRUE)
+	H.fully_replace_character_name(M.name, H.real_name)
+
+	H.remove_all_languages()
+	H.grant_language(/datum/language/sectoid)
```

Comment:
```suggestion
```
A) should have done this on equip instead of ERT creation so adminspawn sectoids also get the language
B) Remove this in favor of a language holder, see the other reviews